drake/examples/Quadrotor/Quadrotor.m

```diff
@@ -113,13 +113,13 @@ function visualizePlan(x,lcmgl)
   methods (Static)
     
     function runOpenLoop
-      r = Quadrotor('lidar');
+      r = Quadrotor();
```

Comment:
I turned this off to stop bullet from complaining.  With it on I was getting this:
```
In BulletModel::findClosestPointsBtwElements: No closest point found!
```